.github/workflows/golint.yml

```diff
@@ -0,0 +1,49 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - run: | 
+          sudo apt-get update 
+          sudo apt-get install libgpgme-dev libgpgme11
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
We'll have to add this as part of the revendoring process so we don't miss this.